src/logic/currencyValues/api/fetchTokenCurrenciesBalances.ts

```diff
@@ -11,9 +11,13 @@ export type BalanceEndpoint = {
   usdConversion: string
 }
 
-const fetchTokenCurrenciesBalances = (safeAddress: string): Promise<AxiosResponse<BalanceEndpoint[]>> => {
+const fetchTokenCurrenciesBalances = (
+  safeAddress: string,
+  excludeSpamTokens = true,
+  onlyTrustedTokens = true,
```

Comment:
Should we add this parameters as environment configuration and activate just on mainnet?

I'm having a strange behaviour with Rinkeby tokens. With this options activated I see 0 balance but I still see the tokens on the list.

Also in Rinkeby most of the tokens are not trusted/validated that's why I ask to activate on mainnet environments

Reply:
@Uxio0 could you please give us some input on this?

Reply:
Maybe is a bug, @dasanra @mikheevm could you provide the Safe you are using for testing?